### ISSUE TITLE
Bule command not working

### ISSUE THREAD

**Issue by seanskggo72** [vale1410/bule#1]
Hello!

So in the instructions for set up, commands such as bule list and bule add is not working. Specifically, the error message is:

-----------------------------------------------------------------
 Command 'bule' not found, did you mean:

  command 'buble' from deb node-buble (0.19.8-8)

Try: sudo apt install <deb name> 

-----------------------------------------------------------------

I know that the commands are stored in cmd directory but how can I install bule to make it source these files?

**Comment by SirZebe** [comment]
1. clone from github repo
2. inside bule directory run 'go build'
3. make sure . (current dir) is in your PATH, run 'export PATH=.$PATH'
4. (optional) add bash auto-completion with TAB key by running 'source
<(bule completion bash 2>/dev/null)'
5. make sure you've got .bule.yaml in your home directory, if not you can
run 'bule add --newconfig' for empty configuration
6. add new solvers like 'bule add depqbf @"--qdo" QBF --label depqbf
--setdefault" (see readme for more information)


sob., 19 gru 2020 o 13:30 seanskggo72 <notifications@github.com> napisał(a):

**Comment by vale1410** [comment]
If you don't want to compile Bule yourself (instead of step 1 and 2 in SirZebe's explanation), you can also download the executables from https://github.com/vale1410/bule/releases/latest

**Comment by seanskggo72** [comment]
Thank you so much! I got everything working now. 
So first of all, I could not find the bule directory in the repo so I used the executable from bule releases instead, It made much more sense after adding the directory to path - made the bule run like a command instead of as an executable. 
For me, I cloned depqbf into the bule directory and added to bule list using:
```bule add --newconfig```
then
```bule add depqbf/depqbf @"--no-dynamic-nenofex --qdo" QBF -l default```

depqbf was the name of the cloned directory and depqbf inside that directory was the executable for the solver. Everything works fine now!

Just a quick question however: do I need to also add caqe or is depqbf by itself enough to solve dimacs files?
